.github/workflows/scripts/supported-versions.sh

```diff
@@ -6,6 +6,10 @@ set -e
 set -x
 
 if git merge-base --is-ancestor origin/main HEAD
+then
+  printf '["%s", "%s"]' v1.30.1-k3s1 v1.32.1-k3s1
+  exit 0
+elif git merge-base --is-ancestor origin/release/v6.0 HEAD
```

Comment:
This is deprecated/unused in main, so perhaps we can remove this script altogether in this PR

Reply:
https://github.com/rancher/backup-restore-operator/pull/639/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133dd